package/src/skia/Skia.ts

```diff
@@ -11,11 +11,12 @@ import type { IRect } from "./Rect";
 import type { IRRect } from "./RRect";
 import type { RuntimeEffectFactory } from "./RuntimeEffect";
 import type { ShaderFactory } from "./Shader";
-import { Color } from "./Color";
+import type { IColor } from "./Color";
+import { processColor } from "./Color";
 import type { IMatrix } from "./Matrix";
 import type { PathEffectFactory } from "./PathEffect";
 import type { IPoint } from "./Point";
-import type { Vertices, VertexMode } from "./Vertices/Vertices";
+import type { IVertices, VertexMode } from "./Vertices/Vertices";
```

Comment:
Same with the type here - just Vertices sounds fine?

IMatrix, Point etc also?